tests/TestUtility/AssertUtils.cs

```diff
@@ -12,7 +12,7 @@ public static void AssertIgnoringIndent(string expected, string actual)
 
         private static string TrimLines(string source)
         {
-            return string.Join("\n", source.Split('\n').Select(s => s.Trim()));
+            return string.Join("", source.Split('\n').Select(s => s.Trim()));
```

Comment:
Any reason for this change? I'd argue readability is better if line breaks are kept in place.

Reply:
There were multiple versions of method with same name, however there are some tests that require that new lines are ignored also. I ended up to add another utility method for those cases `AssertIgnoringIndentAndNewlines` which is now used for those few cases where new lines were important.